index.html

```diff
@@ -17,12 +17,12 @@
       class="flex justify-end w-screen gap-[0.9375rem] pt-[0.625rem] pl-[0.9375rem] pr-[0.9375rem] absolute top-0"
     >
       <p
-        class="font-arial text-black mt-auto mb-auto text-sm hover:underline cursor-pointer"
+        class="font-arial text-black mt-auto mb-auto text-sm hover:underline cursor-pointer nav"
```

Comment:
try to remove that `nav` class , there is no special style for this , use tailwind instead